components/card/index.css

```
@@ -60,196 +188,282 @@ governing permissions and limitations under the License.
-  padding-inline-end: var(--spectrum-card-subtitle-padding-right);
-}
-
-.spectrum-Card-description {
-  font-size: var(--spectrum-card-subtitle-text-size);
+  padding-inline-end: var(--mod-card-subtitle-padding-right, var(--spectrum-card-subtitle-padding-right));
+  color: var(--highcontrast-card-title-color, var(--mod-card-title-color, var(--spectrum-card-title-color)));
+  text-transform: none;
 }
 
 .spectrum-Card-subtitle + .spectrum-Card-description:before {
   content: "•";
-  padding-inline-end: var(--spectrum-card-subtitle-padding-right);
+  padding-inline-end: var(--mod-card-subtitle-padding-right, var(--spectrum-card-subtitle-padding-right));
 }
 
-.spectrum-Card-footer {
-  padding-block-start: var(--spectrum-card-footer-padding-top);
-  margin-inline-end: var(--spectrum-card-body-padding-right);
-  padding-block-end: var(--spectrum-card-body-padding-bottom);
-  margin-inline-start: var(--spectrum-card-body-padding-left);
+.spectrum-Card-description {
+  font-family: var(--mod-card-body-font-family, var(--spectrum-card-body-font-family));
+  font-size: var(--mod-card-body-font-size, var(--spectrum-card-body-font-size));
+  font-weight: var(--mod-card-body-font-weight, var(--spectrum-card-body-font-weight));
+  font-style: var(--mod-card-body-font-style, var(--spectrum-card-body-font-style));
+  line-height: var(--mod-card-body-line-height, var(--spectrum-card-body-line-height));
+  color: var(--highcontrast-card-body-font-color, var(--mod-card-body-font-color, var(--spectrum-card-body-font-color)));
+}
 
-  border-block-start: var(--spectrum-card-footer-border-top-size) solid;
+.spectrum-Card-footer {
+  margin-block-start: var(--mod-card-footer-margin-top, var(--spectrum-card-footer-margin-top));
+  margin-inline-start: var(--mod-card-body-spacing, var(--spectrum-card-body-spacing));
+  margin-inline-end: var(--mod-card-body-spacing, var(--spectrum-card-body-spacing));
+  padding-block-end: calc(var(--mod-card-body-spacing, var(--spectrum-card-body-spacing)) - var(--mod-card-border-width, var(--spectrum-card-border-width)));
+  color: var(--highcontrast-card-body-font-color, var(--mod-card-body-font-color, var(--spectrum-card-body-font-color)));
+  line-height: var(--mod-card-body-line-height, var(--spectrum-card-body-line-height));
 }
 
 .spectrum-Card-header {
   display: flex;
-  align-items: baseline;
+  align-items: center;
 }
 
 .spectrum-Card-actionButton {
   flex: 1;
   align-self: center;
   display: flex;
   justify-content: flex-end;
+  z-index: 1;
 }
 
 .spectrum-Card--quiet {
-  .spectrum-Card-preview {
-    min-block-size: var(--spectrum-card-quiet-min-height);
+  &:focus,
+  &.is-focused {
+    &::after {
+      border-width: 0;
+    }
+    .spectrum-Card-preview {
+      &::after {
+        border-width: var(--mod-card-focus-indicator-width, var(--spectrum-card-focus-indicator-width));
+        border-color: var(--mod-card-focus-indicator-color, var(--spectrum-card-focus-indicator-color));
+      }
+    }
+  }
```

```suggestion
  &:focus::after,
  &.is-focused::after {
    border-width: 0;
  }
  
  &:focus .spectrum-Card-preview::after,
  &.is-focused .spectrum-Card-preview::after {
      border-width: var(--mod-card-focus-indicator-width, var(--spectrum-card-focus-indicator-width));
      border-color: var(--mod-card-focus-indicator-color, var(--spectrum-card-focus-indicator-color));
  }
```

I found this nesting a bit difficult to grok at first - do you think it would be beneficial to pull them out like this for readability/debugging purposes?

---

I felt like it was a lot to. I'll play around with it! Your approach is already easier to read.

---

I added a comment and condensed the preview selector. I felt like since they were both using the focus state I didn't want to split them up entirely. Thoughts?